### ISSUE TITLE
There should be a badge or something for a cancelled Campaign

### ISSUE THREAD

**Issue by RamRamez** [Giveth/giveth-dapp#2375]
When a campaign is cancelled , there is no badge or status info to inform other users that the campaign is not active anymore.

**Comment by divine-comedian** [comment]
I just cancelled my campaign and I visit the campaign URL and I'm informed on the campaign description that it's been cancelled - 
![image](https://user-images.githubusercontent.com/67759413/126520690-8f94294e-51e2-432e-bb51-f198c160255f.png)

this is cool but I also noticed that all my affiliated interactions, donations, delegations, traces that we're associated with the cancelled campaign dissapear - it would be useful to keep these viewable on the dashboard and add a badge informing the user that the recipient Campain or parent Campaign has been cancelled. 

This will help users understand what's happened rather than be confused by seeing their transactions and entities disappear from giveth TRACE. thoughts? @MoeNick
